tests/test_taling_a_vacation.py

```diff
@@ -0,0 +1,63 @@
+import sys
+
+import pytest
+
+from codecademy.taling_a_vacation import hotel_cost
+from codecademy.taling_a_vacation import plane_ride_cost
+from codecademy.taling_a_vacation import rental_car_cost
+from codecademy.taling_a_vacation import trip_cost
+
+@pytest.mark.parametrize("test_hotel_cost_input,test_hotel_cost_expected", [
+    (1, 140),
+    (2, 2*140),
+    (7, 7*140),
+    (31, 31*140),
+    (365, 365*140)
+])
+def test_hotel_cost(test_hotel_cost_input, test_hotel_cost_expected):
+    assert hotel_cost(test_hotel_cost_input) == test_hotel_cost_expected
+
+
+def test_hotel_cost_negative():
```

Comment:
test function name doesn't correspond to the actual test:
name suggests that you will call hotel_cost() with some negative number but you call hotel_cost() with a string.
Well we need both of these tests (negative number and different type of argument (string, boolean, etc.))